tests/acceptance/30_custom_promise_types/cp.sh

```diff
@@ -24,10 +24,18 @@ do_evaluate() {
         response_result="kept"
         return 0
     fi
-    if cp "$request_attribute_from" "$request_promiser"; then
+    if [ "$request_promise_type" = "cpa" ]; then
+      if sleep 1 && cp -a "$request_attribute_from" "$request_promiser"; then
```

Comment:
sleep is probably useful during manual testing, but probably not during automated testing?

Reply:
We need the timestamps to differ, otherwise there's no difference between `cp` and `cp -a` (or `cp -p`).

Reply:
I've added a comment.

Reply:
Do we need to sleep a whole second though, for the timestamps to differ? How about `sleep 0.0000001`?

Reply:
The timestamp is in seconds.